scormxblock/scormxblock.py

```diff
@@ -152,7 +152,7 @@ def updoad_all_content(temp_directory, fs):
 
     if len(all_content) < FILES_THRESHOLD_FOR_ASYNC:
         # We estimate no problem here, just upload the files
-        s3_upload(all_content, temp_directory, dest_dir)
+        s3_upload.delay(all_content, temp_directory, dest_dir)
```

Comment:
Why this change is necessary?

Reply:
It was not running in a external worker. I think now the task has to be called with delay